roles/openshift/tasks/firewalld.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
The tricky part about the sdn is that with the current openshift-sdn-node package is that when using it, you need to manage the openshift-node service through the openshift-sdn-node service.

However, I do agree with your rationale.  The current plan I have is something along the lines of this:
- openshift-common (will include base_os, repos, other common tooling)
  - openshift-master
    1. conditional include for openshift-sdn-master
  - openshift-node
    1. conditional include for openshift-sdn-node
  - openshift-sdn
    - openshift-sdn-node
    - openshift-sdn-master
 ... and so on.